doc/sphinx/Makefile.sphinx

```diff
@@ -63,6 +63,13 @@ html: symlink-docs check-sphinxbuild clean-build
 	@echo
 	@echo "Build finished. The HTML pages are in "'$(BUILDPATH)'"/html."
 
+check: symlink-docs check-sphinxbuild clean-build
```

Comment:
Do we want to clean the build whether or not an error occurred?  Should the client of `make check` be required to clean up the build directory themselves?